ovis/vllm/aimv2/visual_tokenizer_aimv2.py

```diff
@@ -0,0 +1,120 @@
+import PIL
+import torch
+from torch import softmax
+from torch.nn.functional import gumbel_softmax, pad
+from transformers import AutoImageProcessor
+from vllm.config import QuantizationConfig
+from vllm.model_executor.layers.linear import ColumnParallelLinear
+from app.engine.vllm_model.aimv2.modeling_aimv2 import AIMv2Model
+from app.engine.vllm_model.ovis_config import BaseVisualTokenizerConfig
```

Comment:
Note that in the current state, this branch won't work without correcting these imports